firedrake/utility_meshes.py

```diff
@@ -784,6 +784,7 @@ def BoxMesh(nx, ny, nz, Lx, Ly, Lz, reorder=None, distribution_parameters=None,
                  v2, v4, v6, v7,
                  v1, v2, v7, v4]
         cells = np.asarray(cells).swapaxes(0, 3).reshape(-1, 4)
+        raise NotImplementedError("The crossed cutting of hexahedra has a broken connectivity issue for Pk (k>1) elements")
```

Comment:
Thanks, I think there is a test in tests/regression/test_mesh_generation.py as well that needs to be removed.

Reply:
Yes, the test is now removed. Thanks!